src/block/block_ckpt.c

```diff
@@ -661,14 +661,19 @@ static int
 __ckpt_add_blkmod_entry(
   WT_SESSION_IMPL *session, WT_BLOCK_MODS *blk_mod, wt_off_t offset, wt_off_t len)
 {
-    uint64_t end, start;
-    uint32_t end_buf_bytes, end_rdup_bytes;
+    uint64_t end_bit, start_bit;
+    uint32_t end_buf_bytes, end_rdup_bits, end_rdup_bytes;
 
     WT_ASSERT(session, blk_mod->granularity != 0);
-    start = (uint64_t)offset / blk_mod->granularity;
-    end = (uint64_t)(offset + len) / blk_mod->granularity;
-    WT_ASSERT(session, end < UINT32_MAX);
-    end_rdup_bytes = WT_MAX(__wt_rduppo2((uint32_t)end, 8), WT_BLOCK_MODS_LIST_MIN);
+    /*
+     * Figure out how the starting and ending bits based on the granularity and our offset and
+     * length.
+     */
+    start_bit = (uint64_t)offset / blk_mod->granularity;
+    end_bit = (uint64_t)(offset + len) / blk_mod->granularity;
+    WT_ASSERT(session, end_bit < UINT32_MAX);
+    end_rdup_bits = WT_MAX(__wt_rduppo2((uint32_t)end_bit, 64), WT_BLOCK_MODS_LIST_MIN);
```

Comment:
64 because the bitmap grows 8 bytes at a time?  Is it worth providing a constant or comment so this is clear?  (On the other hand, I wasn't particularly puzzled by it.)

Reply:
I added a comment.